AAI/README.md

```diff
@@ -19,160 +19,98 @@ permalink: aai-introduction
 ### Introduction
 
 The [GA4GH AAI profile specification]({% link AAI/AAIConnectProfile.md %})
-leverages OpenID Connect (OIDC) Servers for use in authenticating the identity of
-researchers desiring to access clinical and genomic resources from [data
+leverages OpenID Connect (OIDC) Servers to authenticate researchers
```

Comment:
"Servers" is a bit old school (and possibly not even correct)?? Consider deleting the word?